aspnetcore/security/data-protection/configuration/scaling.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Configure ASP.NET Core Data Protection in distributed or load-balanced environments
+author: acasey
+description: Learn how to configure Data Protection in ASP.NET Core for multi-instance apps.
+ms.author: acasey
+ms.custom: mvc
```

Comment:
@Rick-Anderson Do you happen to know what this means/does?  I just copied it from the overview.